test/e2e/specs/specs-playwright/wp-likes-spec.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+import {
+	BrowserHelper,
+	DataHelper,
+	LoginFlow,
+	NewPostFlow,
+	GutenbergEditorPage,
+	LikesComponent,
+} from '@automattic/calypso-e2e';
+
+/**
+ * Constants
+ */
+const mochaTimeOut = config.get( 'mochaTimeoutMS' );
+const host = DataHelper.getJetpackHost();
+const viewportName = BrowserHelper.getViewportName();
+const quote =
+	'The foolish man seeks happiness in the distance. The wise grows it under his feet.\n— James Oppenheim';
+
+describe( `[${ host }] Likes: (${ viewportName }) @parallel`, function () {
```

Comment:
Just an idea here - if we want to have all those top-level titles consistent, maybe we should have some kind of `createSuiteTitle` helper for that:

```js
// test-helper.js
import { browserHelper, dataHelper } from "@automattic/calypso-e2e";

const host = dataHelper.getJetpackHost();
const viewportName = browserHelper.getViewportName();

function createSuiteTitle( title ) {
  return `[${ host }] ${title}: (${ viewportName }) @parallel`
}
```

```js
// wp-likes-spec.js
import { createSuiteTitle } from 'test-helper';

describe( createSuiteTitle( 'Likes' ), function () { 
  //
} );
```

Reply:
I'd like to hear what @scinos and @bsessions85 think about this.

On one hand, I like reducing boilerplate. It reduces inconsistencies and issues when copy-pasting into new files.
But on the other hand, having a function to generate test name would hide some meaningful information elsewhere in the codebase.

What do others think? Implementation wise, I am thinking it should be like `DataHelper.createSuiteTitle`.

Reply:
`DataHelper.createSuiteTitle` sounds good to me 👍

Reply:
I will file a follow-up task to address this.

Reply:
This is an interesting approach. It is certainly something we could try and see how it looks. It would definitely create consistency.

Reply:
I've implemented the follow-up PR for the suite title: #53244 .